DTAdmin/Subjects/AddNewRecordViewController.swift

```diff
@@ -0,0 +1,94 @@
+//
+//  AddNewRecordViewController.swift
+//  DTAdmin
+//
+//  Created by ITA student on 10/18/17.
+//  Copyright © 2017 if-ios-077. All rights reserved.
+//
+
+import UIKit
+
+class AddNewRecordViewController: UIViewController {
+
+    private var httpStatusCode : Int? = nil
+    
+    let queryService = QueryService()
+    
+    @IBOutlet weak var subjectName: UITextField!
+    
+    @IBOutlet weak var subjectDescription: UITextView!
+    
+    var updateDates = false
+    
+    var subjectId : String = ""
```

Comment:
Please put a colon just after a variable name (e.g. subjectId: String). Do the same for the rest.